src/client/components/EmployersQuestions/index.js

```diff
@@ -184,18 +187,17 @@ function EmployerQuestions(props) {
           value={stateFuncs.reason.get}
           onChange={(e) => handleChange("reason", e.target.value)}
         >
-          {["still-working", "completed", "fired", "quit", "laid-off"].map(
-            (reason) => (
-              <option key={reason} value={reason}>
-                {t(tk("reason-" + reason))}
-              </option>
-            )
-          )}
+          {["fired", "quit", "laid-off", "still-working"].map((reason) => (
```

Comment:
Note that the default is "still-working" (from the newEmployerData() function above). I assume that's still correct even though it's the last option?

Reply:
@nguyen-tina do you have a preference for the default option?

Reply:
Still working works!